source/slang/slang.natvis

```diff
@@ -842,4 +842,81 @@
     <Type Name="Slang::BasicExpressionType">
         <DisplayString>BasicExpressionType ({*(DeclRefBase*)m_operands.m_buffer[0].values.nodeOperand})</DisplayString>
     </Type>
+
+	<Type Name="Slang::CapabilitySet">
+		<DisplayString>{m_targetSets.map.m_values}</DisplayString>
+		<Expand>
+			<CustomListItems>
+				<Item Name="m_targetSets">m_targetSets</Item>
+				<Item Name="m_targetSets.map.m_values">m_targetSets.map.m_values</Item>
+			</CustomListItems>
+		</Expand>
+	</Type>
+	<Type Name="Slang::CapabilityTargetSet">
+		<DisplayString>{{target={target}}}</DisplayString>
+		<Expand>
+			<CustomListItems>
+				<Item Name="target">target</Item>
+				<Item Name="shaderStageSets">shaderStageSets</Item>
+				<Item Name="shaderStageSets.map.m_values">shaderStageSets.map.m_values</Item>
+			</CustomListItems>
+		</Expand>
+	</Type>
+	<Type Name="Slang::CapabilityStageSet">
+		<DisplayString>{{size={disjointSets.count}}}</DisplayString>
+		<Expand>
+			<CustomListItems>
+				<Item Name="stage">stage</Item>
+				<Item Name="disjointSets">disjointSets</Item>
+			</CustomListItems>
+			<Synthetic Name="disjointSets.[data]">
+				<Expand>
+			        <LinkedListItems>
+				        <HeadPointer>disjointSets.head</HeadPointer>
+				        <NextPointer>next</NextPointer>
+				        <ValueNode>value</ValueNode>
+			        </LinkedListItems>
+				</Expand>
+			</Synthetic>
+		</Expand>
+	</Type>
+	<Type Name="Slang::UIntSet">
+		<DisplayString>{{max_size={m_buffer.m_count*Slang::UIntSet::kElementSize}}}</DisplayString>
```

Comment:
This will be very useful. I wonder we should just create a 
struct CapabilityAtomSet : UIntSet
and create the visualizer on just CapabilityAtomSet.